[Excel] (Custom Functions) Adding coauthoring note (#742)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-overview.md

}
```

## Co-Authoring

> Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature is supported by custom functions. If your workbook uses a custom function, your sharing partner will be prompted to install the custom function's add-in in order to display the results. Once both people sharing the document have installed the add-in, all custom function behavior should work normally.

Reviewer: ```suggestion
Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature works with custom functions. If your workbook uses a custom function, your colleague will be prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function will share results through co-authoring.
```

Reviewer: Just a few consistency recommendations like using "colleague" similar to support docs like here https://support.office.com/en-us/article/collaborate-on-excel-workbooks-at-the-same-time-with-co-authoring-7152aa8b-b791-414c-a3bb-3024e46fb104

> For more information on co-authoring, see [About CoAuthoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).

Author: ```suggestion
For more information on co-authoring, see [About Co-Authoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
```

> For more information on coauthoring, see [About CoAuthoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).

Reviewer: ```suggestion
For more information on co-authoring, see [About CoAuthoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
```
